Добавлены скрипты миграции Liquibase

timurash opened the pull request:
https://github.com/timurash/timeTrackingService/issues/19#issue-688022049

timurash committed:
Добавлены скрипты миграции Liquibase и дравйвер для PostgreSQL

https://github.com/timurash/timeTrackingService/issues/19#issue-688022049

timurash committed:
В файл Readme.md добавлена информация по скриптам миграции

AlexandrYZ commented on Readme.md:
```diff
@@ -15,4 +15,11 @@
+
+# Инструкции
+
+## Миграции
```
Какой командой обновить БД?

AlexandrYZ reviewed (approved):
Действительно ли нужен в репозитории этот файл - migrations/liquibaseDrivers/postgresql-42.2.16.jar
Возможно ли его скачать из источника? По какой ссылке?

AlexandrYZ commented on Readme.md:
```diff
@@ -15,4 +15,11 @@
+# Инструкции
+
+## Миграции
+В репозитории имеются скрипты миграции Liquibase для создания таблиц и добавления в них тестовых данных.
```
Загрузку тестовых данных удобнее реализовать из CSV с помощью [loadData](https://docs.liquibase.com/change-types/community/load-data.html).

timurash committed:
Отредоктирован файл Readme.md из репозитория удален JDBC Driver

https://github.com/timurash/timeTrackingService/pull/20#pullrequestreview-478337488
https://github.com/timurash/timeTrackingService/pull/20#discussion_r479912602

timurash committed:
Теперь тестовые данные загружаются из CSV файлов

https://github.com/timurash/timeTrackingService/pull/20#discussion_r479914746
